ovn-tester/ovn_tester.py

```diff
@@ -193,9 +193,11 @@ def create_nodes(cluster_config, central, workers):
     ] if cluster_config.clustered_db else [
         'ovn-central'
     ]
+    relay_containers = []
     for i in range(cluster_config.n_relays):
-        db_containers.append(f'ovn-relay-{i + 1}')
-    central_node = CentralNode(central, db_containers, mgmt_net, mgmt_ip)
+        relay_containers.append(f'ovn-relay-{i + 1}')
```

Comment:
Nit: wouldn't it be a bit more "pythonic" as below?
```
relay_containers = [f'ovn-relay-{i + 1}' for i in range(cluster_config.n_relays)]
```

Reply:
I changed this based on your suggestion.